CMakeLists.txt

```diff
@@ -1,19 +1,57 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.8)
 
-project(dbg_macro)
+project(dbg_macro LANGUAGES CXX)
 
-set(CMAKE_CXX_STANDARD 11)
+###############################################################################
```

Comment:
can we please remove these long `####` lines? A single
```
# CMake options
```
comment is good enough, I think.

Reply:
Sure!

Reply:
Done!